src/sp_execute.c

```diff
@@ -123,6 +124,22 @@ static void sp_execute_ex(zend_execute_data *execute_data) {
     return;
   }
 
+  if ((SNUFFLEUPAGUS_G(config).config_sloppy->enable)) {
+    zend_op* orig_opline = (void*)execute_data->opline;
```

Comment:
Why the cast in `(void*)` ?

Reply:
I could cast to (zend_op*) but i need to discard the const qualifier.

Reply:
Cast to `(const zend_op*)` ?

Reply:
I modify `orig_opline` during the loop.

Reply:
mh, indeed:

```C
struct _zend_execute_data {
	const zend_op       *opline;           /* executed opline                */
	zend_execute_data   *call;             /* current call                   */
	zval                *return_value;
	zend_function       *func;             /* executed function              */
	zval                 This;             /* this + call_info + num_args    */
	zend_execute_data   *prev_execute_data;
	zend_array          *symbol_table;
#if ZEND_EX_USE_RUN_TIME_CACHE
	void               **run_time_cache;   /* cache op_array->run_time_cache */
#endif
};
```

I don't think that you're supposed to modify this member then :D